Crash in proper_plugin.py when one argument passed to `isinstance`

**Crash Report**

When using the `misc/proper_plugin.py` plugin, mypy crashes when only one argument is passed to `isinstance`

**Traceback**

```
test.py:1: error: Too few arguments for "isinstance"
test.py:1: error: INTERNAL ERROR -- Please try using mypy master on Github:
https://mypy.readthedocs.io/en/stable/common_issues.html#using-a-development-mypy-build
Please report a bug at https://github.com/python/mypy/issues
version: 0.910
Traceback (most recent call last):
  File "mypy/checkexpr.py", line 3905, in accept
  File "mypy/checkexpr.py", line 271, in visit_call_expr
  File "mypy/checkexpr.py", line 353, in visit_call_expr_inner
  File "mypy/checkexpr.py", line 858, in check_call_expr_with_callee_type
  File "mypy/checkexpr.py", line 917, in check_call
  File "mypy/checkexpr.py", line 1029, in check_callable_call
  File "mypy/checkexpr.py", line 729, in apply_function_plugin
  File "/mnt/e/Pranav/Github/mypy/misc/proper_plugin.py", line 37, in isinstance_proper_hook
    right = get_proper_type(ctx.arg_types[1][0])
IndexError: list index out of range
test.py:1: : note: use --pdb to drop into pdb
```

**To Reproduce**

test.py:
```py
 isinstance(5)
```

Run mypy with `mypy test.py --show-traceback` (or just `mypy test.py`).

**Your Environment**

- Mypy version used: 0.910
- Mypy command-line flags: None necessary
- Mypy configuration options from `mypy.ini` (and other config files):

mypy.ini:
```ini
[mypy]
# Change this path if you're not at the root of the mypy repo
plugins = misc/proper_plugin.py
```

- Python version used: 3.8.5
- Operating system and version: Ubuntu 20.04.2 LTS on WSL

--- a/misc/proper_plugin.py
+++ b/misc/proper_plugin.py
@@ -34,6 +34,9 @@ class ProperTypePlugin(Plugin):
 
 
 def isinstance_proper_hook(ctx: FunctionContext) -> Type:
+    if len(ctx.arg_types) != 2 or not ctx.arg_types[1]:
+        return ctx.default_return_type
+
     right = get_proper_type(ctx.arg_types[1][0])
     for arg in ctx.arg_types[0]:
         if (is_improper_type(arg) or

